pages/price-feeds/fetch-price-updates.mdx

```diff
@@ -0,0 +1,76 @@
+    ]
+  }
+}
+```
+
+## SDK
+
+Pyth provides a typescript SDK to fetch price updates. This SDK allows you to read the real-time price updates to consume it on- and off-chain applications.
+
+```typescript
+const connection = new PriceServiceConnection("https://hermes.pyth.network", {
+  priceFeedRequestConfig: {
+    // Provide this option to retrieve signed price updates for on-chain contracts.
+    // Ignore this option for off-chain use.
+    binary: true,
+  },
+});
+
+const priceIds = [
+  // You can find the IDs of prices at https://pyth.network/developers/price-feed-ids
+  "0xe62df6c8b4a85fe1a67db44dc12de5db330f7ac66b72dc658afedf0f4a415b43", // BTC/USD price id
+  "0xff61491a931112ddf1bd8147cd1b641375f79f5825126d665480874634fd0ace", // ETH/USD price id
+];
+
+// Get the latest price updates for the given price IDs.
+// If you set `binary: true` above, then this method also returns signed price updates for the on-chain Pyth contract.
+const priceUpdates = await connection.getLatestVaas(priceIds);
+```
+
+## Websocket
```

Comment:
sorry i should have mentioned this earlier but the endpoint isn't websocket anymore. It's actually Server-Sent Events. I think you could make this section heading "Streaming" instead.